src/components/Terminal/Terminal.js

```diff
@@ -47,7 +48,12 @@ const Terminal = ({ address, modelName }) => {
       const mousePosition = e.clientY;
       const newTerminalHeight =
         viewPortHeight - mousePosition + terminalHeaderHeight;
-      if (newTerminalHeight >= minimumTerminalHeight) {
+
+      const maximumTerminalHeight = viewPortHeight - modelDetailHeaderHeight;
+      if (
+        newTerminalHeight < maximumTerminalHeight &&
+        newTerminalHeight >= minimumTerminalHeight
```

Comment:
Instead of this you could use `max` and `min` when setting the `newTerminalHeight` variable. For example:

``` js
const proposedTerminalHeight = viewPortHeight - mousePosition + terminalHeaderHeight;
const newTerminalHeight = Math.min(Math.max(proposedTerminalHeight, minimumTerminalHeight), maximumTerminalHeight);
```

Reply:
@anthonydillon Yes, that would also work but in terms of preferring human-readable code, I'd prefer to stick with what I've got?